### PR TITLE
Added simple release scripts

### PR DESCRIPTION
**Pull request by edurdevic**

### CODE REVIEW

File: scripts/tag_version.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import os
```
Comment (milos-colic) [inline scripts/tag_version.py:1]: @edurdevic can we add examples of usage to CONTRIBUTING.md?

File: scripts/tag_version.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import sys
+import re
+
+def main(version):
+    os.system("git checkout main && git pull")
+    os.system("git tag -a v_%s -m 'Version %s'" % (version, version))
```
Comment (milos-colic) [inline scripts/tag_version.py:7]: f"" string would look neater here maybe - but this works as well

File: scripts/tag_version.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    os.system("git push origin v_%s" % version)
+
+if __name__ == "__main__":
+    if len(sys.argv) < 2:
+        raise Exception("Please specify a version. Use format x.y.z.")
+    version = sys.argv[1]
+
+    if (not re.match(r'\d+\.\d+\.\d+', version)):
```
Comment (milos-colic) [inline scripts/tag_version.py:15]: I thought the version has "vX.Y.Z" format with "v" being required. Is this still the case?
If yes I would add it to the match pattern then drop it (will stabilise the regex a bit more).

**Review by milos-colic — CHANGES_REQUESTED** [review]
Can we just please add description of this to CONTRIBUTING.md?

File: scripts/tag_version.py (hunk 2 above)
Comment (edurdevic) [reply scripts/tag_version.py:7]: Yes, the fact is the Copilot wrote it and I was too lazy to change it ;P 
Let me refactor

File: scripts/tag_version.py (hunk 3 above)
Comment (edurdevic) [reply scripts/tag_version.py:15]: When calling the script you do not need to specify the `v`. The script will add it only where required

**Review by milos-colic — APPROVED** [review]

**Pull request merged**